amp-conformance/Tests/4_Basic_Data_Elmnts/4_2_extent/4_2_3_m/tile/Negative/pad.01/test.cpp

```diff
@@ -11,6 +11,7 @@
 #include <amptest.h>
 #include <amptest_main.h>
 #include <string>
+#include <climits>
```

Comment:
Yes, these issues are exposed by the libstdc++ headers due to variations in the implementations and also they are less tolerant to these small bugs in the tests